azure-functions-shared-codegen/src/lib.rs

```diff
@@ -28,11 +27,9 @@ use syn::{parse, ItemMod};
 /// #[generated_mod]
 /// mod MyModule {}
 /// ```
-#[proc_macro_attribute]
-pub fn generated_mod(_: TokenStream, input: TokenStream) -> TokenStream {
-    let m = parse::<ItemMod>(input).unwrap();
-
-    let ident = &m.ident;
+#[proc_macro]
+pub fn generated_mod(item: TokenStream) -> TokenStream {
+    let ident = parse::<Ident>(item).unwrap();
```

Comment:
Why the change to `Ident` here?  This macro is parsing a `mod` declaration, which is what `ItemMod` represents.  I think `Ident` would get you `mod` or a failure (assuming quote checks for identifiers that conflict with keywords), no?

Reply:
Oh I didn't notice you also changed it to a non-attribute macro.  I'm good with this 👍.